.internal/baseOrderBy.js

```diff
@@ -15,7 +16,12 @@ import isArrayLike from '../isArrayLike.js'
 function baseOrderBy(collection, iteratees, orders) {
   let criteriaIndex = -1
   let eachIndex = -1
-  iteratees = iteratees.length ? iteratees : [(value) => value]
+  iteratees = iteratees.length ? iteratees.map((iteratee) => {
+    if (Array.isArray(iteratee)) {
+      return (value) => baseGet(value, iteratee)
+    }
+    return iteratee
+  }) : [(value) => value]
```

Comment:
We have a method called `identity` that can me used for `(value) => value`.

Reply:
@jdalton the existing identity was defined in ../test/util.js, hence i defined a identity function in this file as there is no other util file, let me know if it makes sense.